css/pages/cart-page.css

```diff
@@ -0,0 +1,208 @@
+.hero-heading {
+  margin: 3rem 0;
+  font-size: 2.5rem;
+  text-align: center;
+  font-weight: 600;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+}
+
+.cart-container {
+  display: flex;
+  justify-content: center;
+  gap: 10rem;
+}
+
+.order__container {
+  width: 30rem;
+  padding: 2rem;
+  background-color: var(--card-bg-color);
+  position: sticky;
+  height: min-content;
+  top: calc(78px + 3rem);
+}
```

Comment:
if possible try to use rem here insted px; rest all looks good to me.

Reply:
sure thank you